amxmodx/amxmodx.cpp

```diff
@@ -2860,8 +2860,13 @@ static cell AMX_NATIVE_CALL set_user_flags(AMX *amx, cell *params) /* 3 param */
 
 	if (id > 31)
 		id = 31;
-
-	pPlayer->flags[id] |= flag;
+		
+	if(params[4])
```

Comment:
You need to check for parameter availability here.

Reply:
Sorry, I'm fairly new to this. How do I exactly add that check?

Reply:
Here's an example
https://github.com/alliedmodders/amxmodx/blob/7b3646a012e4cc330c17328e13b31fec6a3280a8/amxmodx/amxmodx.cpp#L1443

Reply:
Is it actually necessary? The 4th parameter has a default value of "false" if it's not used. Just curious why it's needed.

Reply:
@OciXCrom you will need to recompile all plugins using `get_players`.

Reply:
@OciXCrom Plugins that don't get compiled against the new *.inc will pass you 3 parameters, and core needs to handle that situation gracefully.

Reply:
@Nextra @Amaroq7 @WPMGPRoSToTeMa - Thanks, I added the necessary check.